package/src/findCsvFiles.js

```diff
@@ -11,6 +11,8 @@ async function findCsvFiles(dirPath) {
 
   // eslint-disable-next-line no-shadow
   async function helper(dirPath) {
+    if (dirPath.includes('.json') || dirPath.includes('.md')) return;
```

Comment:
i don't like these short versions. make it casual

Reply:
I will make it as regex pattern

Reply:
ok